frontends/web/src/routes/bitsurance/account.tsx

```diff
@@ -97,16 +104,17 @@ export const BitsuranceAccount = ({ code, accounts }: TProps) => {
           <Header title={<h2>{t('bitsuranceAccount.title')}</h2>} />
           <View width="550px" verticallyCentered fullscreen={false}>
             <ViewContent>
-              { btcAccounts.length === 0 ? (
+              { groupedOptions.length === 0 ? (
                 <div>{t('bitsuranceAccount.noAccount')}</div>
               ) : (
-                <AccountSelector
+                <GroupedAccountSelector
                   title={t('bitsuranceAccount.select')}
-                  options={btcAccounts}
                   disabled={disabled}
+                  options={groupedOptions}
                   selected={selected}
                   onChange={handleChangeAccount}
```

Comment:
here you didn't rewrite onChange, but in the 2 other similar components you have  `onChange={(selected) => setSelected(selected)}` and `onChange={setCode}`.

Reply:
oups, good point. this one comes from the Bitsurance staging merge,  I'll just change this to setState. Same with the () => setSelected, will change to setSelected. Ty!